src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
+
+	if( m_saveIntervalSlider->value() != value )
+	{
+		m_saveIntervalSlider->setValue( value );
+	}
+
+	m_saveIntervalLbl->setText( tr( "Auto save interval: %1 %2" ).arg(
```

Comment:
I take back that statement... yeah this is just as versatile either way, just potentially confusing, perhaps a short [translator comment](http://doc.qt.io/qt-4.8/i18n-source-translation.html#translator-comments) would suffice. :+1: